package.json

```diff
@@ -43,24 +43,34 @@
   "author": "Netlify Inc.",
   "license": "MIT",
   "dependencies": {
-    "@babel/core": "^7.11.4",
-    "@babel/preset-env": "^7.11.5",
-    "@rollup/plugin-babel": "^5.2.0",
-    "@rollup/plugin-commonjs": "^21.0.0",
-    "@rollup/plugin-inject": "^4.0.2",
+    "@babel/core": "^7.16.7",
+    "@babel/preset-env": "^7.16.7",
+    "@rollup/plugin-babel": "^5.3.0",
+    "@rollup/plugin-commonjs": "^21.0.1",
+    "@rollup/plugin-inject": "^4.0.4",
     "@rollup/plugin-json": "^4.1.0",
-    "@rollup/plugin-node-resolve": "^11.0.0",
-    "@types/node": "^16.0.0",
+    "@rollup/plugin-node-resolve": "^13.1.3",
+    "@types/node": "^16.11.19",
     "buffer-es6": "^4.9.3",
     "del": "^6.0.0",
-    "make-dir": "^3.1.0",
-    "node-fetch": "^2.6.1",
+    "node-fetch": "^2.6.6",
     "path-type": "^4.0.0",
     "process-es6": "^0.11.6",
-    "rollup": "^2.23.1",
+    "rollup": "^2.63.0",
     "rollup-plugin-node-polyfills": "^0.2.1",
     "rollup-plugin-terser": "^7.0.2",
-    "typescript": "^4.1.5"
+    "typescript": "^4.5.4"
+  },
+  "devDependencies": {
+    "@netlify/build": "^26.1.0",
+    "@netlify/eslint-config-node": "^4.0.6",
+    "ava": "^3.15.0",
+    "c8": "^7.11.0",
+    "date-fns": "^2.28.0",
+    "husky": "^4.3.8",
```

Comment:
I kept husky although it's already included in `@netlify/eslint-config-node`. Not sure what's your policy is.